srcpkgs/neomutt/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'neomutt'
 pkgname=neomutt
-version=20210205
-revision=2
+version=20211029
+revision=1
 wrksrc="neomutt-${version}"
```

Comment:
we can remove this line

Reply:
that doesn't seem to work well. the pkg doesn't build with that one and `build_wrksrc="$wrksrc"` a couple lines later removed:

```
/void-packages/common/build-style/configure.sh: line 8: ./configure: No such file or directory
=> ERROR: neomutt-20211029_1: do_configure: '${configure_script} ${configure_args}' exited with 127
=> ERROR:   in do_configure() at common/build-style/configure.sh:8
```

Reply:
nevermind, I forgot that this package has multiple distfiles

Reply:
We should still be able to remove this line if we replace `build_wrksrc="$wrksrc"` with `build_wrksrc="$pkgname-$version"`